digdag-core/src/main/java/io/digdag/core/agent/CheckedConfig.java

```diff
@@ -62,7 +45,7 @@ public Config deepCopy()
     @Override
     public ObjectNode getInternalObjectNode()
     {
-        this.usedKeys.setAll(true);
+        this.usedKeys.setAllUsed(true);
```

Comment:
The purpose of `CheckedConfig` class is showing `Did you mean ...?` message when some local config keys are not used. CheckedConfig tracks that which keys are accessed by the operator and which keys are not.
CheckedConfig assumes that all keys are accessed when `getInternalObject` method is called.